scene/gui/color_picker_shape.cpp

```diff
@@ -122,7 +122,7 @@ void ColorPickerShape::draw_sv_square(Control *p_control, const Rect2 &p_square,
 		Vector2(p_square.position.x, end.y),
 	};
 
-	Color color1 = color_picker->color;
+	Color color1 = color_picker->_get_pick_color_normalized();
```

Comment:
```suggestion
	Color color1 = color_picker->color_normalized;
```
Shapes have access to picker's private members.